src/redux/products/products.reducer.js

```diff
@@ -25,12 +25,14 @@ export const initialState = {
   isHotItem: true,
```

Comment:
Is it a duplication of the `isHotItemFilter` from `filters` object

Reply:
deleted